.travis.yml

```diff
@@ -96,7 +98,8 @@ before_install:
     if [[ "$UNSUPPORTED" == true ]]; then
       pip install 3to2
       export PIP_NO_COMPILE=1
-      sed -E -i.bak '/^(coala|mypy-lang)/d' requirements.txt bear-requirements.txt
+      sed -E -i.bak '/^(coala|mypy-lang)/d' requirements.txt
```

Comment:
Oh, my `'/^(coala|mypy-lang)/d'` trick has come back to haunt me.
It was intended to only invoke `sed` once, but it has two different operations occurring here.

Change this to

> sed -i.bak '/^coala/d' requirements.txt